scripts/benchmarks-ci.sh

```diff
@@ -1,70 +1,25 @@
 #!/usr/bin/env bash
 
-steps=50
-repeat=20
 category=$1
 runtimeName=$2
 artifactsDir=$3
+steps=${4:-50}
+repeat=${5:-20}
 
 benchmarkOutput=./parachains/runtimes/$category/$runtimeName/src/weights
 benchmarkRuntimeName="$runtimeName-dev"
```

Comment:
@muharem 
what about renaming `benchmarkRuntimeName` -> `benchmarkRuntimeChain` ?

Reply:
runtimes have names. chains are run by nodes and run runtimes.